app/src/modules/location/location.c

```diff
@@ -22,13 +22,10 @@
 LOG_MODULE_REGISTER(location_module, CONFIG_APP_LOCATION_LOG_LEVEL);
 
 BUILD_ASSERT(CONFIG_APP_LOCATION_WATCHDOG_TIMEOUT_SECONDS >
-			 CONFIG_APP_LOCATION_TRIGGER_TIMEOUT_SECONDS,
```

Comment:
Nit: Some alignment issues here it looks like. Could align with CONFIG_APP_LOCATION